pkg/name/digest.go

```diff
@@ -45,14 +46,31 @@ func (d Digest) Identifier() string {
 	return d.DigestStr()
 }
 
+// TagStr returns the tag component of the Digest.
+func (d Digest) TagStr() string {
+	return d.tag
```

Comment:
It looks like this is only tested through `Name()`, perhaps add some focused tests?

Reply:
Probably a good idea, even though it was just a simple accessor method.